src/test/java/in/zapr/druid/druidry/extensions/datasketches/postAggregator/ThetaSketchEstimatePostAggregatorTest.java

```diff
@@ -55,8 +55,10 @@ public void testRequiredFieldsWithFieldAccess() throws JsonProcessingException,
                 new FieldAccessPostAggregator("stars");
 
         ThetaSketchEstimatePostAggregator thetaSketchEstimatePostAggregator =
-                new ThetaSketchEstimatePostAggregator("estimate_stars", fieldAccessPostAggregator);
-
+                ThetaSketchEstimatePostAggregator.builder()
```

Comment:
Remove additional spaces

Reply:
In imports, extra lines are due to auto reformat code style. And I have removed in other parts.